tests/ttnn/unit_tests/operations/backward/test_backward_fill.py

```diff
@@ -57,8 +58,63 @@ def test_bw_fill_opt_tensor(input_shapes, device):
     assert len(pages_before) == len(ttnn._ttnn.reports.get_buffer_pages())
 
     golden_function = ttnn.get_golden_function(ttnn.fill_bw)
-    golden_tensor = golden_function(grad_data, in_data)
+    golden_tensor = golden_function(grad_data, in_data, value=2.0)
 
     tt_output_tensor_on_device = [input_grad]
     comp_pass = compare_all_close(tt_output_tensor_on_device, golden_tensor, atol=0, rtol=0)
     assert comp_pass
+
+
+# Pytorch Reference
+#    name: fill.Tensor(Tensor self, Tensor value) -> Tensor
+#    self: zeros_like(grad)
+#    value: grad.sum()
+#    result: at::fill(self_t, value_t)
+@pytest.mark.parametrize(
+    "input_shapes",
+    (
+        (torch.Size([1, 1, 32, 32])),
```

Comment:
Please add tests for shape which is not divisible by 32.
and 3rd dimension equal to 1.

Reply:
only TILE layout is used in the test as the implementation uses ttnn.add which accepts TILE layout tensors.